jenkinsfiles/Jenkinsfile.grobid

```diff
@@ -18,7 +20,7 @@ elifePipeline {
         stage 'Push unstable image', {
             def image = DockerImage.elifesciences(this, 'grobid', commit)
             def unstable_image = image.addSuffixAndTag('_unstable', commit)
-            unstable_image.tag('latest').push()
+            unstable_image.tag(tag).push()
```

Comment:
If I put in a commit as the revision, this would then try to push the same commit twice. Not sure if that matters?

Reply:
Should be idempotent, yeah.

Reply:
But supports (accidentally) using a branch.